app/models/lesson.rb

```diff
@@ -1,3 +1,5 @@
+require "duration"
```

Comment:
[`require_dependency`](https://apidock.com/rails/ActiveSupport/Dependencies/Loadable/require_dependency) would possibly be better here if there is an autoloading issue